test/specs/startOf.spec.js

```diff
@@ -1,110 +1,72 @@
 const {DateTime} = luxon;
 
-describe('Luxon Adapter', function() {
-  it('should accept timezone configuration', function() {
-    var chart = acquireChart({
-      type: 'line',
-      data: {
-        datasets: [{
-          data: [{
-            x: 0,
-            y: 100
-          }]
-        }]
-      },
-      options: {
-        scales: {
-          x: {
-            type: 'time',
-            time: {
-              unit: 'second',
-              displayFormats: {
-                second: 'MMM d, yyyy, h:mm:ss a'
-              }
-            },
-            adapters: {
-              date: {
-                zone: 'UTC+6'
-              }
-            },
-            ticks: {
-              source: 'data'
-            }
-          }
-        }
-      }
-    });
-
-    expect(chart.scales.x.ticks[0].label).toEqual('Jan 1, 1970, 6:00:00 AM');
-  });
-
-  it('should parse Luxon DateTime objects directly', function() {
-    var adapter = new Chart._adapters._date();
-
-    expect(adapter.parse(DateTime.fromMillis(0))).toEqual(0);
-    expect(adapter.parse(DateTime.fromISO('1970-01-01T00:00:01Z'))).toEqual(1000);
-  });
-
-  it('should parse Luxon Date objects directly', function() {
-    var adapter = new Chart._adapters._date();
-
-    expect(adapter.parse(new Date(0))).toEqual(0);
+describe('\'startOf\' method', function() {
```

Comment:
you could probably avoid the need for escaping by switching up the quoting method:
```
`'startOf' method`
```

Reply:
done

Reply:
LINT doesn't like it:

 3:10  error  Strings must use singlequote  quotes

Reply:
ah, my other projects use the `{ avoidEscape: true }` option with the eslint rule: https://eslint.org/docs/latest/rules/quotes#avoidescape